pages/lists/[listId]/index.tsx

```diff
@@ -1,5 +1,18 @@
-const ListsHub = () => {
-  return <div>Redirect to Lists Overview</div>;
+import { GetServerSidePropsContext } from "next";
+
+const ListIndex = () => {
+  return <></>;
 };
 
-export default ListsHub;
+export function getServerSideProps(context: GetServerSidePropsContext<{ listId: string }>) {
+  const { listId } = context.params!;
+
+  return {
```

Comment:
Can a redirect not be used instead of keeping `pages/lists/[listId]/index.tsx`?

Reply:
Good call. Changed to a redirect in the next config